src/components/modal/modal.tsx

```diff
@@ -304,7 +304,6 @@ export class Modal implements ConditionalSlotComponent {
   @Method()
   async setFocus(focusId?: "close-button"): Promise<void> {
     const closeButton = this.closeButtonEl;
-
```

Comment:
Sorry, I missed this. Can you restore this? It's an unrelated change.

Reply:
ahh, my bad. will restore this.